src/firmware/host/mod.rs

```diff
@@ -102,46 +107,58 @@ impl Firmware {
                 0 => State::Uninitialized,
                 1 => State::Initialized,
                 2 => State::Working,
-                _ => return Err(Indeterminate::Unknown),
+                _ => return Err(FirmwareError::KnownSevError(SevError::InvalidPlatformState))?,
```

Comment:
```suggestion
                _ => return Err(SevError::InvalidPlatformState)?,
```

Can't we just do this?